lib/plugins/track.js

```diff
@@ -205,6 +205,10 @@ exports.configure = function configure (config) {
 
   // Values must be initialized later because they depend on knowing the uuid
   _.defaults(this.client.track.values, configureValues(this.client.track))
+  this.setCookie = setCookie
```

Comment:
Not a great fan of this one, but since we're refactoring in the near future i didn't think too much of it